.github/workflows/ci.yml

```diff
@@ -125,7 +125,12 @@ jobs:
           name: coverage-${{ matrix.python }}-${{ matrix.airflow }}
           path: ./.coverage
 
-  Run-example-dag-tests-Airflow-2-5:
+  Run-example-dag-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ '3.8', '3.9']
```

Comment:
In some areas it seems we are calling this `{{matrix.version}}` (e.g line 111), in others `{{matrix.python}}`, should we normalise, perhaps sticking to `matrix.python`?

Reply:
yes, a great catch, thank you!